VisualC/SDL/SDL.vcxproj.filters

```diff
@@ -537,9 +537,6 @@
     <ClInclude Include="..\..\src\events\SDL_events_c.h">
       <Filter>events</Filter>
     </ClInclude>
-    <ClInclude Include="..\..\src\events\SDL_eventfilter_c.h">
-      <Filter>events</Filter>
```

Comment:
Why remove filter ?

Reply:
I guess the MSVC project file changes should only be addition of new SDL_hidapi_8bitdo.c source.

Now that you reverted MSVC project file changes, they can be done later after @slouken reviews the code.